frontend/app/features/settings/pages/workflows/workflow-step.component.html

```diff
@@ -62,15 +62,51 @@
             </div>
         </div>
 
+        <div class="row transition-prevent-updates no-guttersw">
+            <div class="col col-step" style="max-width: 12rem">
         <div class="form-check">
-            <input class="form-check-input" type="checkbox" id="preventUpdates_{{step.name}}"
+                    <input class="form-check-input" style="margin-top: 0.8rem;" type="checkbox" id="preventUpdates_{{step.name}}"
                 [disabled]="disabled"
                 [ngModel]="step.noUpdate"
                 (ngModelChange)="changeNoUpdate($event)" />
 
             <label class="form-check-label" for="preventUpdates_{{step.name}}">
                 Prevent updates
             </label>
-        </div> 
+    </div>
+            </div>
+            <ng-container *ngIf="step.noUpdate">
+                <div class="col-auto col-label">
+                    <span class="text-decent">when</span>
+                </div>
+                <div class="col col-step-expression">
+                    <input class="form-control" [class.dashed]="!step.noUpdateExpression" spellcheck="false"
```

Comment:
Please do not align with the tag names, just a normal tab

Reply:
Sure, I've updated the files. These files were updated in VS 2019, so default settings were applied. Going forward, we'll be using VS code so it will remain sorted.